Track-your-savings-goals/index.js

```diff
@@ -0,0 +1,7 @@
+const startingBalance = 700;
+const montlyContribution = 200;
+const months = 12;
+
+totalSavings = startingBalance + (montlyContribution * months)
+
+console.log("After 12 months of saving, you will have", totalSavings ,"in your account");
```

Comment:
On this line, it's important to change the hard coded 12 for the real amount of months, coming from the variable. You can make that using somethin called template literals, like this:

console.log(After ${months} months of saving, you will have ${totalSavings} in your account);